test/userController.test.js

```diff
@@ -94,3 +94,143 @@ describe('GET /api/v1/users/:user_id endpoint', () => {
     expect(response.status).toBe(404);
   });
 });
+
+describe('PATCH /api/v1/users/updateCurrentUser endpoint', () => {
+  it('should update the current user', async () => {
+    const signedCookie = await loginAndReturnCookie({
+      email: 'ava@ava.com',
+      password: 'secret',
+    });
+
+    const updatedUserData = {
+      email: 'new_email@example.com',
+      name: 'new_name',
+    };
+
+    const response = await request(app)
+      .patch('/api/v1/users/updateCurrentUser')
+      .send(updatedUserData)
+      .set('Cookie', [signedCookie]);
+
+    expect(response.status).toBe(200);
+    expect(response.body).toHaveProperty('user');
+    expect(response.body.user).toMatchObject(updatedUserData);
+  });
+
+  it('should return an error if required data is missing', async () => {
+    const signedCookie = await loginAndReturnCookie({
+      email: 'ava@ava.com',
+      password: 'secret',
+    });
+
+    const response = await request(app)
+      .patch('/api/v1/users/updateCurrentUser')
+      .set('Cookie', [signedCookie]);
+
+    expect(response.status).toBe(400);
+    expect(response.body).toHaveProperty('msg', 'Please provide all values');
+  });
+});
+
+describe('GET /api/v1/users/getCurrentUserWithPurchasedAlbums endpoint', () => {
+  it('should return the data of the currently authenticated user with purchased albums', async () => {
+    // Act: Log in and get a signed cookie
+    const signedCookie = await loginAndReturnCookie(testUserCredentials);
+
+    // Act: Get the current user's information with purchased albums using the authenticated cookie
+    const response = await request(app)
+      .get('/api/v1/users/getCurrentUserWithPurchasedAlbums')
```

Comment:
```suggestion
      .get('/api/v1/users/showMe/withAlbums')
```

Reply:
You can also verify this by checking the backend API docs and verifying the correct URL needed to be called in this case